fideslog/api/database/csv_writer.py

```diff
@@ -0,0 +1,37 @@
+import csv
+from uuid import uuid1
+
+from ..schemas.analytics_event import AnalyticsEvent
+
+
+class Pipe:
+    """
+    Class that imitates a file object in write mode
+    Used to write out a csv for use with S3
+    """
+
+    value = ""
+
+    def write(self, text: str) -> None:
+        """
+        Sets the value to be returned as a csv
+        """
+        self.value = self.value + text
+
+
+def file_name_random() -> str:
+    """
+    Generates a random uuid to be passed as the filename
+    """
+    return uuid1().hex + ".csv"
```

Comment:
This might be a job for a later story, but it would be cool if we could instead structure the  file name by timestamp (of course, S3 also logs date modified, so this isn't a big deal). This way if we ever needed to troubleshoot something with the flushing process, we could find files more easily. 

Depending on the level of traffic we're expecting, and based on the date-based folder, I'd recommend `hh-mm.csv`

Reply:
Left the uuid as well but files will now be formatted as `hh-mm-*.csv` 👍🏽